src/components/counter/counter.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './counter.css';
+class Counter extends React.Component {
+    constructor(props){  
+        super(props)    
+        this.state = {count : 0}  
+        this.incrementCount = this.incrementCount.bind(this)
+        this.decrementCount = this.decrementCount.bind(this) 
+      }  
+        
+      incrementCount(){  
+        let i=this.state.count;
+        i++;
+        this.setState({count : i})  
```

Comment:
Remove line 12, 13. 
Please use callback function with setState, replace line 14 with below.
`
this.setState(prevState => {
   return {count: prevState.count + 1}
})
`

Reply:
Fixed it.

Reply:
Please use without return if the statement is a single line.
`this.setState((prevState) => ({ count: prevState.count + 1 }));`